lessons/Algebra2/exploring-linear-models/langs/en-us/pages/model-college-v-income.adoc

```diff
@@ -0,0 +1,67 @@
+</style>
+++++
+
+For this page, you'll need to the @starter-file{alg2-states} open on your computer. If you haven't already, select **Save a Copy** from the "File" menu to make a copy of the file that's just for you.
+
+== Build a Model from Samples
+
+**Make a scatter plot showing the relationship between `college-or-higher` and `median-household-income`.**
+
+@n Record the `college-or-higher` and `median-household-income` values for Alabama (AL) and Alaska (AK), as (x,y) pairs below:
+
+@center{
+ (@fitbruby{10em}{@ifsoln{22.6}}{AK college-or-higher}, @fitbruby{10em}{@ifsoln{43253}}{AK median-household-income}) @hspace{10em} (@fitbruby{10em}{@ifsoln{27.5}}{AL college-or-higher}, @fitbruby{12em}{@ifsoln{70760}}{AL median-household-income})
+}
+
+@n Using the two points above, compute the slope and y-intercept to define a linear model (in Function Notation and Pyret) by filling in the blanks below. Show your work!
+
+@vspace{12ex}
+
+@center{
+ @math{f(x) =} @fitbruby{8em}{@ifsoln{5613.67}}{slope}x + @fitbruby{8em}{@ifsoln{-83616.02}}{y-intercept} @hspace{5em} @show{(code '(define (f x) (+ (* (?ANS 5613.67) x) (?ANS -83616.02))))}
+}
+
+@n Type your function into the Definitions Area on the left, and click "Run". Make sure you fix any errors or warnings. In the Interactions Area, try plugging in the `college-or-higher` value for Alabama by typing `f(22.6)`. Does it closely predict the right median income for Alabama? What about Alaska? Consider: If it doesn't predict it perfectly, why might that be?
+
+@n Try plugging in different `college-or-higher` values from _other_ states, to see how well a model based only on Alabama and Alaska fits the rest of the country. For what state does this model work well? @fitb{5em}{} For what state does it work poorly? @fitb{5em}{}
+
+== Fit the Model
+
+**Evaluate @show{(code '(fit-model states-table "state" "college-or-higher" "median-household-income" f))} in the Interactions Area.**
+
+@n In the upper left corner, you'll see that it says "R-sqr", followed by a number. What is that number? @fitb{}{@ifsoln{-15.63}}
+
+@n Change the definition of `f` so that the slope is _negative_ and click "Run". What is the @math{R^2} value now? @fitb{}{}
+
+@n Change the definition of `f` so that it draws a horizontal line. What is the @math{R^2} value? @fitb{}{}
+
+@n What do you think @math{R^2} tells us? @fitb{}{@ifsoln{It measures how well the model fits the data}}
+
+== Build a Model through Trial {amp} Error
+
+@n Try using Massachusetts (row 21) and Nevada (row 28) instead of Alabama and Alaska. What @math{R^2} do you get? @fitb{}{@ifsoln{2.84}}
+
+@n What is the **best @math{R^2} you can get**, if the line didn't need to pass through two specific states? @fitb{5em}{} Write the model below:
+
+@center{
+ @math{f(x) =} @fitbruby{8em}{}{slope}x + @fitbruby{8em}{}{y-intercept} @hspace{5em} @show{(code '(define (f x) (+ (* (?ANS "will vary") x) (?ANS "will vary"))))}
+}
+
+== Build a Model Computationally
+
```

Comment:
Perhaps we should state more directly that "lr-plot computes the optimal linear model using all of the data points"

Reply:
Done